### DISCUSSION TITLE
Add ThanosRuler as a possible datasource in Grafana Alerting

### DISCUSSION THREAD

**Discussion by migueljfsc** [grafana/grafana#64432] [Alerting]
I'd like to be able to configure Grafana Alerting to query my ThanosRuler instance instead of an AlertManager instance.

I'm currently monitoring multiple kubernetes clusters, each having their own alertmanager instances to manage alerts related to their own cluster.
To have an aggregated view of all alerts, I have a ThanosRuler instance that queries all alertmanagers. However, there is no way to use it here:
<img width="1221" alt="image" src="https://user-images.githubusercontent.com/25540481/223755098-ee315486-8064-4e22-8bd8-29d81d420105.png">

Since it only allows Prometheus Alertmanager sources, and I've been unable to configure one pointing to Thanos-Ruler, I am unable to view all my alerts in a single place.

The obvious workaround is adding a datasource for each of my alertmanagers and then keep switching between datasources to view queries, but this is not desirable

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82792. Feel free to continue the discussion around this there. Thank you!
